infrastructure/demo-release.bat

```diff
@@ -1,5 +1,5 @@
 CALL cd ..
```

Comment:
I'd move this set of comamnds to single bat.
And then jst have kind **demo.bat** which says **relase.bat demo-covid19livebot**
inside release bat it would be
```CALL heroku container:login
CALL heroku container:push web --app=%1
CALL heroku container:release web --app=%1
CALL heroku open --app=%1
```

Reply:
@danbilokha

Reply:
letme try

Reply:
done